exonum/src/node/basic.rs

```diff
@@ -42,22 +43,26 @@ impl NodeHandler {
             Ok(Any::Block(msg)) => self.handle_block(&msg),
             Ok(Any::Transaction(msg)) => self.handle_tx(msg),
             Err(err) => {
-                error!("Invalid message received: {:?}", err.description());
+                error!(
+                    self.network_logger(),
+                    "Invalid message received: {:?}",
+                    err.description()
+                );
             }
         }
     }
 
     /// Handles the `Connected` event. Node's `Connect` message is sent as response
     /// if received `Connect` message is correct.
     pub fn handle_connected(&mut self, addr: SocketAddr, connect: Connect) {
-        info!("Received Connect message from peer: {}", addr);
+        info!(self.network_logger(), "Received Connect message";  "peer_address" => %addr);
         self.handle_connect(connect);
     }
 
     /// Handles the `Disconnected` event. Node will try to connect to that address again if it was
     /// in the validators list.
     pub fn handle_disconnected(&mut self, addr: SocketAddr) {
-        info!("Disconnected from: {}", addr);
+        info!(self.network_logger(), "Disconnected message";  "peer_address" => %addr);
```

Comment:
I wonder that we have a some architecture issues if we need a `network_logger` here.